examples/job.py

```diff
@@ -1,3 +1,6 @@
+import sys, os
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
```

Comment:
@leore @pon 
This looks kinda hacky but it will load the unpacked version before anything else. Is there a better way of doing it? It's completely dependent on the relative path of the file.